.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +0,0 @@
-### Zenhub Link:
```

Comment:
I think you may need to update your branch to include the latest commits from `develop`. This file should be in there.